.github/workflows/yaml_and_ansible_linting.yml

```diff
@@ -0,0 +1,34 @@
+name: "Run ansible-lint and yamllint for this repo."
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push events and pull_request
+  #push:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: "Ansible-lint"
+      # Install ansible-lint(5.0.6) together with yamllint(1.26.0) and run the linting on the whole repo
+        run: |
+          sudo python -m pip install ansible
```

Comment:
I think `sudo` is not needed here. You are running as `root`.

Reply:
Yep, just tested it locally, there is no need for sudo to be here.